pkgs/development/compilers/rust/default.nix

```diff
@@ -4,6 +4,7 @@
 , bootstrapVersion
 , bootstrapHashes
 , selectRustPackage
+, ...
```

Comment:
I would rather just remove the `rustcPatches` attribute from the 1.44 file instead of adding this here.